DeveloperGuide.md

```diff
@@ -143,6 +143,132 @@ Attributes are driven by user input. No predetermined fields aside from the part
 The dependencies for the backend lambda functions are handled using poetry. If you changed the lambda functions make sure to do a `cdk deploy` to reflect the change.
 
 The lambda handlers are categorized based on the project domain. E.g you will find all assets related functions in `/backend/backend/assets` folder.
+
+# Adding your own pipelines
+When you create pipelines in VAMS, you have two options
+1. Create a lambda pipeline
+2. Create a sagemaker pipeline
+
+## Lambda pipeline
+
+Lambda pipelines are piplelines that use AWS Lambda as their compute platform.
+
+When you create a lambda pipeline you can either allow VAMS to create a new lambda function  or provide a name of an existing lambda function to be used as a pipeline.
```

Comment:
Might want to always say `VAMS Lambda Pipeline` and `AWS Lambda` or just `Lambda` wherever we use these terms for clarity sake around the proper nouns.